src/foundry/foundry.js/pixi/containers/canvasLayers/placeablesLayer.d.ts

```diff
@@ -327,35 +366,6 @@ declare global {
      *                (unused)
      */
     protected _onDeleteKey(event?: any): void;
-
```

Comment:
With those methods removed, `ConfiguredDocumentClass` is unused and can be removed from the imports.